pyaedt/modeler/cad/Primitives.py

```diff
@@ -4924,6 +4924,70 @@ def import_spaceclaim_document(self, input_file):
         self.refresh_all_ids()
         return True
 
+    def discovery_import(self, input_file):
+        """Import a Discovery file.
+
+        Parameters
+        ----------
+        input_file :
+            Full path and name of the Discovery file.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oEditor.CreateUserDefinedModel
+        """
+        version = settings.aedt_version[-4:].replace(".", "")
+
+        aedt_install_dir = os.environ.get("ANSYSEM_ROOT{}".format(version), "")
+        ansys_install_dir = os.environ.get("AWP_ROOT{}".format(version), "")
+
+        if ansys_install_dir == "":
+            self.logger.error("Same version of AEDT and Discovery is needed.")
```

Comment:
Here it seems that you are checking if ansys unified is installed.
I think we could modify the message to be:

```suggestion
        if ansys_install_dir == "":
            self.logger.error("Ansys is not installed. Consequently, Discovery installation is missing too.")
```

Reply:
Thanks @MaxJPRey for your comment! AEDT can be installed independently from Discovery, so my intention is to check if the same version of AEDT and Disco are installed. However it's possible that I'm not doing it correctly.

Reply:
You are doing it properly @IreneWoyna . I am just suggesting to modify the error message.
I think we should just mention in the log file which version of Ansys is missing, such as:
`self.logger.error("Ansys {} is not installed. Consequently, Discovery installation is missing too.".format(version))`

Reply:
I agree with @MaxJPRey, but the situation is a bit trickier. A user can have the correct Ansys version installed, but without having Discovery installed on the system (Discovery can be deselected in the installation). So my suggestion is to check if Disco is installed (rather than Ansys unified) and log the message accordingly, just about the Discovery version missing.

Reply:
You are correct @Alberto-DM.